content/en/news/security/istio-security-2021-001/index.md

```diff
@@ -2,7 +2,7 @@
 title: ISTIO-SECURITY-2021-001
 subtitle: Security Bulletin
 description: JWT authentication can be bypassed when AuthorizationPolicy is misused.
-cves: [N/A]
+cves: [CVE-2021-21378]
```

Comment:
Normally we have the score details, AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:L/A:N, here as well.